src/ui/components/Button/Button.scss

```diff
@@ -30,6 +30,15 @@
   );
 }
 
+// This is intended for counter-actions like cancelling or undoing, etc.
+.Button--counter-action {
```

Comment:
I get the notion that this is "counter" to the original action but I think that's a bit of unclear phrasing, especially as people might read this as a number "Counter" (which could also have a button).

"cancel" or "undo" would be a better name.

Reply:
I think counter-action is more versatile since it's generic enough to convey the concept of cover cancel, dismiss or undo but I can change it, no problem.